vp_functions.py

```diff
@@ -492,66 +498,44 @@ def altitude_parameter_averaging_cvp_static(radar, field, cvp_index, avg_range_d
         rhohv = rhohv_3D_data[np.ix_(range(0,radar.nsweeps),az_indxs,r_indxs)]
         (meteoMask) = kdpfun.generate_meteo_mask(elev, rays, bins, flags, rhohv, METEO_THRESH)
         column = kdpfun.unwrap_phidp(elev, rays, bins, meteoMask, column)
-
-    if verbose:
-        print ("column.shape is {}".format(column.shape) )
-    #summed = np.zeros(column.shape)
-    counted = np.zeros(column.shape)
-    mask = np.ma.masked_invalid(column).mask
-    inv_mask = np.where(mask, 0, 1)
-    #summed = np.nansum(column, axis=1)
-    counted = np.nansum(inv_mask, axis=1)
-    observation_count = np.where(counted == 0, np.nan, counted)
-    mean_values = np.nanmean(column, axis=1)
-    std_values = np.nanstd(column, axis=1)
+    
```

Comment:
I will assume that this code is scientifically correct, as that is your area of expertise @rrniii